ipamd/ipamd.go

```diff
@@ -232,16 +246,18 @@ func (c *IPAMContext) reconcileENIIP() {
 		log.Debugf("Attempt again to allocate IP address for eni :%s", eni.Id)
 		err := c.awsClient.AllocAllIPAddress(eni.Id)
 		if err != nil {
-			ipamdErrInc("reconcileENIIPAllocAllIPAddressFailed", err)
+			ipamdErrInc("retryAllocENIIPAllocAllIPAddressFailed", err)
 			log.Warn("During eni repair: error encountered on allocate IP address", err)
 			return
 		}
 		ec2Addrs, _, err := c.getENIaddresses(eni.Id)
 		if err != nil {
-			ipamdErrInc("reconcileENIIPgetENIaddressesFailed", err)
+			ipamdErrInc("retryAllocENIIPgetENIaddressesFailed", err)
 			log.Warn("During eni repair: failed to get ENI ip addresses", err)
 			return
 		}
+		curTime := time.Now()
+		c.lastNodeIPPoolAction = curTime
```

Comment:
Why not just `c.lastNodeIPPoolAction = time.Now()`?

Reply:
done